quilt/tools/command.py

```diff
@@ -148,6 +148,12 @@ def _parse_package(name):
             raise ValueError
     except ValueError:
         raise CommandException("Specify package as owner/package_name.")
+
+    try:
+        PackageStore.check_name(owner, pkg)
+    except StoreException as ex:
+        raise CommandException(str(ex))
```

Comment:
the message should tell people how to do things right. so "Package name contains illegal characters. Rename package with alphanumerics and underscore..." or whatever. I'm not sure if we support numerics in trailing positions.